Prace_domowe/Praca_domowa4/Grupa1/Kacper_Grzymkowski/PD4.ipynb

```diff
@@ -0,0 +1,2439 @@
+{
```

Comment:
<p>Normalizer normalizuje kolejne obserwacje a więc w wierszach, a MinMax normalizuje zmienne (kolumny), a więc to nie są dwie inne metody robienia tego samego.</p><p> </p><p>Normalizer jest częśto używany w unsupervised learning (rzadko w supervised)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>